script/zlib.decompress.py

```diff
@@ -0,0 +1,27 @@
+import zlib
+import sys
+import os
+
+CHUNKSIZE = 1024
+
+if len(sys.argv) != 4:
+    print("Invalid arguments supplied 3 arguments expected")
+    exit(1)
+
+if not os.path.isfile(sys.argv[1]):
+    print("File {0} not exists".format(sys.argv[1]))
+    exit(1)
+
+try:
+  zlib_obj = zlib.decompressobj(int(sys.argv[2]))
+  with open(sys.argv[3], 'wb') as decompressed_file:
+      with open(sys.argv[1], 'rb') as compressed_file:
+          while True:
+              buf = compressed_file.read(CHUNKSIZE)
+              if not buf:
+                  break
+              decompressed_data = zlib_obj.decompress(buf)
+              decompressed_file.write(decompressed_data)
+          decompressed_file.write(zlib_obj.flush())
+except Exception as e:
+  print("Chyba při dekompresi souboru {0}: {1}".format(sys.argv[1], str(e)))
```

Comment:
```suggestion
  print(str(e))
  exit(1)
```

no ten exit je tam zatracene dulezitej, jinak to vyprodukuje 0b soubor a nic to nenapise - coz je na odstrel, prihod tam na to test at se to nekdy nestane znovu

Reply:
Přidal sem unit test pro ten zlib decompressor, je tam test na úspěšnou dekompresi a na fail když je špatně window size. Předopkládám, žes to tak asi myslel.

Reply:
aha no ne :/ myslel jsem jen test na to, ze to failne - treba jak je `invalid-file` test

Reply:
no ne ze bych mel neco proti tem unittestum, ale asi bych je napsal jinak - na manipulaci s tempem pouzivame https://github.com/keboola/php-temp a stejne by to bylo lepsi to udelat pres ty datadir testy (ktery jsou ovsem v tomhle procesoru ve starsi verzi), at je to vsude stejny. Ale to aktualizovat, to je zase zarovka a vyprd bych se na to.

Bud bych to vratil zase zpatky a pridal tam jednoduchej test ve stylu `invalid-file`, ze ten zlib pri spatym souboru failne, anebo bych teda do tech phpunit testu pridal aspon ten Temp at to neni uplne custom

Reply:
Jasně, udělal sem update teď funkcionální testy používají `keboola/datadir-tests` a unit testy `keboola/php-temp`. Nevím jestli tam ten unit test nechat nebo ne, sice je prakticky duplicitní proti tomu data dir testu, ale dává smysl :)

Reply:
jo, nechal bych je tam, nicemu to neskodi :)